### DISCUSSION TITLE
What is the best place to see a list of available images?

### DISCUSSION THREAD

**Discussion by tracymiranda** [wolfi-dev/os#70] [Q&A] [answered]
And bonus if that place lists the sizes of the images...

**Comment by jdolitsky** [comment, marked as answer]
This should ideally grow to include all community images built using Wolfi.

For the Chainguard-provided ones, we can see the ones listed on the org page here: https://github.com/chainguard-images

**Comment by erikaheidi** [comment]
An update to this question: we now have a full reference documentation for Chainguard Images at Academy: https://edu.chainguard.dev/chainguard/chainguard-images/reference/
